src/components/character-count/index.md

```diff
@@ -67,7 +67,9 @@ Do this by setting `data-maxwords` in the component markup. For example, `data-m
 
 When using a character count, try to set the limit higher than most users will need. Find out what this is by doing user research and data analysis.
 
-If the limit is much higher than most users are likely to reach, you can choose to only display the message after a user has entered a certain amount.
+#### If the limit is far higher than users are likely to reach
+
+You can choose to display a character count message only after the user has entered a certain number of characters. This is useful when a character limit is needed, but users are unlikely to reach the limit.
```

Comment:
Technically, it's not about the user entering characters, it's down to a threshold being passed. For example, if the text area is pre-filled, that'll activate the threshold condition as well.
```suggestion
You can choose to display a character count message only after the length of the text has surpassed a certain 'threshold' number of characters. This is useful when a character limit is needed due to the technical limitations of the service, but users are unlikely to reach the limit.
```

Reply:
This might be a teensy bit misleading now as it implies that the threshold is set based on the number of characters, when it's actually a percentage of the total limit.

Reply:
How about just removing the word 'number' entirely then?
> You can choose to display a character count message only after the length of the text has surpassed a certain 'threshold' of characters. This is useful when a character limit is needed due to the technical limitations of the service, but users are unlikely to reach the limit.

Reply:
We might as well go all the way and explain what the threshold is:

```suggestion
You can choose to display a character count message only after the length of text has passed a certain 'threshold', set as a percentage of the total character limit. This is useful when a character limit is needed due to the technical limitations of the service, but users are unlikely to reach that limit.```

Reply:
This sentence is meant to be a bit of preamble before explaining explaining the threshold in the following sentence.